### PR TITLE
Change underway to pending

### PR DESCRIPTION
**Pull request by technosophos**
This addresses https://github.com/cnabio/cnab-spec/pull/334 for the Go library, changing `underway` to `pending`.

Signed-off-by: Matt Butcher <matt.butcher@microsoft.com>

### CODE REVIEW

**Comment by radu-matei** [comment]
The claims marshaling test seems to be failing:

```
--- FAIL: TestMarshal_AllFields (0.00s)
       	            	-
        	Test:       	TestMarshal_AllFields
        	Messages:   	marshaled claim does not match expected
```

File: claim/testdata/claim.allfields.json [hunk 1]
```diff
@@ -1 +1 @@
-{"schemaVersion":"v1.0.0-WD","installation":"my_claim","revision":"revision","created":"1983-04-18T01:02:03.000000004Z","modified":"1983-04-18T01:02:03.000000004Z","bundle":{"schemaVersion":"schemaVersion","name":"mybun","version":"v0.1.0","description":"this is my bundle","invocationImages":null},"result":{"message":"result message","action":"install","status":"underway"},"parameters":{"myparam":"myparamvalue"},"outputs":{"myoutput":"myoutputvalue"},"custom":["anything goes"]}
\ No newline at end of file
+{"schemaVersion":"v1.0.0-WD","installation":"my_claim","revision":"revision","created":"1983-04-18T01:02:03.000000004Z","modified":"1983-04-18T01:02:03.000000004Z","bundle":{"schemaVersion":"schemaVersion","name":"mybun","version":"v0.1.0","description":"this is my bundle","invocationImages":null},"result":{"message":"result message","action":"install","status":"pending"},"parameters":{"myparam":"myparamvalue"},"outputs":{"myoutput":"myoutputvalue"},"custom":["anything goes"]}
```
Comment (vdice) [inline claim/testdata/claim.allfields.json]: Oh, oops, I think a newline was added, causing a test failure.  (Can also update the test expectation instead, if preferred.)

Comment (technosophos) [reply claim/testdata/claim.allfields.json]: I think I will have to. It appears that my editor keeps inserting the newline on save.

Comment (technosophos) [reply claim/testdata/claim.allfields.json]: Fixed in test so that it is no longer sensitive to newlines at the end of the file

**Pull request merged**